tests/rxtx/tests.c

```diff
@@ -181,7 +182,44 @@ static void test_rxtx_rxtx_to_rxtx(void)
 	}
 	pg_graph_destroy(g);
 }
+static void mytx_empty(struct pg_brick *brick,
+                 struct pg_rxtx_packet *tx_burst,
+                 uint16_t *tx_burst_len,
+                 void *private_data)
+{
+	struct pg_rxtx_packet empty[PG_MAX_PKTS_BURST];
```

Comment:
do we really need empty ?
I tx_burst_len is equal 0, rxtx_poll should not even use tx_burst.